tavla/app/components/ContactForm.tsx

```diff
@@ -0,0 +1,93 @@
+'use client'
+import { TextArea, TextField } from '@entur/form'
+import { Label, Paragraph } from '@entur/typography'
+import { SubmitButton } from 'components/Form/SubmitButton'
+import { useFormState } from 'react-dom'
+import { postForm } from './actions'
+import { getFormFeedbackForField } from 'app/(admin)/utils'
+import { useEffect, useRef, useState } from 'react'
+import { FormError } from 'app/(admin)/components/FormError'
+import { useToast } from '@entur/alert'
+import { Expandable } from './Expandable'
+
+export function ContactForm() {
+    const formRef = useRef<HTMLFormElement>(null)
```

Comment:
Jeg antar vi setter en `ref` fordi vi vil klare å nullstille formen, men i følge react kan vi heller gjøre det via en `key`: https://react.dev/learn/preserving-and-resetting-state#resetting-a-form-with-a-key

Reply:
fjernet ref, skal funke nå med usestate